src/app/shared/components/sidenav-filters/sidenav-filters.component.html

```
@@ -1,3 +1,3 @@
-<mat-drawer class="mobile-sidenav" mode="side" *ngIf="isMobileView" [(opened)]="visibleFiltersSidenav">
+<mat-drawer class="mobile-sidenav" mode="side" *ngIf="isMobileView" [opened]="isFiltersSidenavOpen$ | async">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)